package-support/preinstall.sh

```diff
@@ -1,5 +1,5 @@
 
 if ! id -u sideboard &>/dev/null; then
-    groupadd --force -r sideboard
-    useradd -r --shell /sbin/nologin --gid sideboard sideboard
+    groupadd --force -r sideboard -g 600
+    useradd -r --shell /sbin/nologin  -uid 600 --gid sideboard sideboard
```

Comment:
The devops guys at my work say that it's useful if the ``sideboard`` user and group have consistent ids across all of our systems.  That sounded reasonable, so we hard-coded this number.  I also doubt that anyone will use this preinstall script anyway (MAGFest deploys out of Github checkouts).